packages/core/src/common/classes.ts

```diff
@@ -39,15 +39,18 @@ export const INTENT_WARNING = intentClass(Intent.WARNING);
 export const INTENT_DANGER = intentClass(Intent.DANGER);
 
 // text utilities
-export const TEXT_MUTED = "pt-text-muted";
-export const TEXT_OVERFLOW_ELLIPSIS = "pt-text-overflow-ellipsis";
 export const UI_TEXT = "pt-ui-text";
-export const UI_TEXT_LARGE = "pt-ui-text-large";
 export const RUNNING_TEXT = "pt-running-text";
-export const RUNNING_TEXT_SMALL = "pt-running-text-small";
 export const MONOSPACE_TEXT = "pt-monospace-text";
+export const TEXT_LARGE = "pt-text-large";
```

Comment:
if we're changing class names, do we need to create an upgrade script like 1.0 -> 2.0 ?

Reply:
yes probably